Source/Core/DolphinQt/DolphinEmu.entitlements

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need anything here for USB access?

Reply:
Nope, there isn't a special permission for that.